### ISSUE TITLE
Colors have changed between 0.6.6 and 0.6.7

### ISSUE THREAD

**Issue by rhymes** [buefy/buefy#980]
The default color scheme changed between 0.6.6 and 0.6.7.

I've installed 0.6.7, refreshed the app and most icons, bars and link colors have changed

### Overview of the problem

**Buefy** version: [0.6.7]
**Vuejs** version: [2.5.17]
**OS/Browser**: Firefox 62, Chrome 

### Description

The default color scheme changed from 0.6.6 to 0.6.7.

Bulma's version is still the same:

```diff
-buefy@0.6.6:
-  version "0.6.6"
-  resolved "https://registry.yarnpkg.com/buefy/-/buefy-0.6.6.tgz#e59df5258ad254498b888cd612bb1a4cc8773d0a"
+buefy@0.6.7:
+  version "0.6.7"
+  resolved "https://registry.yarnpkg.com/buefy/-/buefy-0.6.7.tgz#739045a3e2437c3abe0a3bcf3694da45f12ee24e"
   dependencies:
-    bulma "^0.7.1"
+    bulma "0.7.1"

-bulma@^0.7.1:
+bulma@0.7.1:
   version "0.7.1"
   resolved "https://registry.yarnpkg.com/bulma/-/bulma-0.7.1.tgz#73c2e3b2930c90cc272029cbd19918b493fca486"
```

### Steps to reproduce

* install Buefy 0.6.7
* change `import 'buefy/lib/buefy.css'` to `import 'buefy/dist/buefy.css'`
* restart the app

### Expected behavior

This is what the various colors looked like with 0.6.6:

![clipboard066](https://user-images.githubusercontent.com/146201/45208801-c7f94300-b28b-11e8-8b61-7f0691f192f0.png)

![download066](https://user-images.githubusercontent.com/146201/45208810-cdef2400-b28b-11e8-8df8-0ceb92df86ff.png)

![search066](https://user-images.githubusercontent.com/146201/45208811-d0ea1480-b28b-11e8-848a-c776ff62c0da.png)

### Actual behavior

This is how they look like as soon as I upgrade to 0.6.7:

![clipboard067](https://user-images.githubusercontent.com/146201/45208842-de9f9a00-b28b-11e8-89cc-827be314ad3a.png)

![download067](https://user-images.githubusercontent.com/146201/45208858-e3fce480-b28b-11e8-9300-ef705e46a9fb.png)

![search067](https://user-images.githubusercontent.com/146201/45208870-e8c19880-b28b-11e8-98a6-4c564ab27532.png)

**Comment by davedoesdev** [comment]
It seems `dist/buefy.css` is using the bulma primary colour (turquoise, `#00d1b2`).
However, `src/scss/utils/_variables.scss` is still setting it to purple, `#7957d5`

**Comment by davedoesdev** [comment]
I ran `npm build:lib` and the generated `dist/buefy.css` had the correct primary color (purple, #7957d5).

Looks like something went wrong when publishing the npm package.

**Comment by davedoesdev** [comment]
Looks like https://github.com/buefy/buefy/commit/64aea4ce918ab0f24e5617a9bbbf105f92b10053#diff-364971b54f78b61e1497715998914b84 fixed it so should be fixed in next version :)

**Comment by rhymes** [comment]
Thank you @davedoesdev !

**Comment by jtommy** [comment]
Fix 0.7.0
